issues/clangcuda/issue-shuffle.cpp

```diff
@@ -0,0 +1,70 @@
+///
+/// Source file containing clangcuda __shfl/__shfl_sync issue reproducer.
+///
+/// Immediate Issue: Warp reductions using shuffle intrinsics give wrong results.
+///
+/// Larger Issue: RAJA reducer tests fail.
+///
+/// Seen when compiling for sm_60 on Sierra EA systems (IBM p8, Nvidia P100)
+/// Seen when compiling for sm_70 on Sierra systems (IBM p9, Nvidia V100)
```

Comment:
Does this reproduce on TOSS3?

Reply:
It didn't reproduce on hasgpu when I tried it.

Reply:
That's interesting. Thanks for the update. Have you tried on pascal?

Reply:
Yes, it does reproduce on manta. I didn't put up a host config as I'm using my own build of clang trunk